### DISCUSSION TITLE
Remove Mac prefix

### DISCUSSION THREAD

**Discussion by JavierSegoviaCordoba** [Chozzle/compose-macos-theme#3] [General]
Compose is build thinking in you can implement your own theme, if you check Material, it doesn't add `Material` prefix to all Material component.

**Comment by Chozzle** [comment]
If I don't add a prefix, it would clash with Material components. All the Mac components are wrapping Material components, so dropping the prefix could be confusing. Generally other blogs have been suggesting a prefix for themed components I think

**Comment by JavierSegoviaCordoba** [comment]
Ah, I thought you was using the basic pack. I think Material should be hidden to the user so the prefix should not be necessary.

Maybe this can be an interesting topic to talk about in Slack 🤔

**Comment by JavierSegoviaCordoba** [comment]
[Link to Slack thread](https://kotlinlang.slack.com/archives/C01D6HTPATV/p1611400432043900?thread_ts=1611380929.040600&cid=C01D6HTPATV)

**Comment by Chozzle** [comment]
So, after the input in slack, I'm still leaning towards using Mac prefixes. I will trial adding WindowsButton soon, and then just Button, that will switch between the two. The main reasons I want to call it MacButton are:

* It describes exactly what it is. This doesn't fall into the same category as Material, where Button from material package could implement a different design system in the future.
* Some consumers might want to use this library as their default design components, but for large apps I would recommend wrapping MacButton in their own Button. That way they can easily change their design system (from Mac to Material to whatever) without being so coupled to this library.
* When I introduce WindowsButton, it could reduce confusion, rather than relying on import package name
